src/main/java/swervelib/motors/SparkMaxBrushedMotorSwerve.java

```diff
@@ -188,8 +175,7 @@ public void setLoopRampRate(double rampRate)
    * @return Motor object.
    */
   @Override
-  public Object getMotor()
-  {
+  public Object getMotor() {
```

Comment:
Is there a more appropriate superclass? Returning as `Object` is potentially dangerous (unless the superclass requires this)

Reply:
I think anything in the `swervelib` folder is part of the YAGSL library, which we have copied here and modified slightly for our needs. I assume 99% of it is its original code.

Reply:
This is the original code from YAGSL